lsp-mode.el

```diff
@@ -1144,7 +1144,7 @@ INHERIT-INPUT-METHOD will be proxied to `completing-read' without changes."
       ;; server may send character position beyond the current line and we
       ;; should fallback to line end.
       (let ((line-end (line-end-position)))
-        (if (> character (- line-end (point)))
+        (if (and (not (equal nil character)) (> character (- line-end (point))))
```

Comment:
could be simply (not character) instead of (not (equal nil character)). This shouldn't affect the other code since it is extending the contract.

Reply:
Hmm - it would have to be ```(not (not character))```, that's not that much better, is it? Let me check if simply ```character``` does work.

Reply:
It does - I'll mend the PR